podcasts/AuthenticationHelper.swift

```diff
@@ -65,7 +65,9 @@ class AuthenticationHelper {
         ServerSettings.refreshToken = response.refreshToken
 
         // we've signed in, set all our existing podcasts to be non synced
-        DataManager.sharedManager.markAllPodcastsUnsynced()
+        if ServerSettings.lastSyncTime == nil {
+            DataManager.sharedManager.markAllPodcastsUnsynced()
```

Comment:
@geekygecko Before merging this I'd like to double-check with you as I might be missing any user case.

The app will only mark all podcasts as unsynced if the user has never logged in to any account in this device.

Reply:
If they have just signed in, the last sync should be nil already. 🤔
If they signed out for six months and subscribed to many podcasts, then signed in again, won't that mean those new podcasts won't appear on their other devices?

Reply:
> won't that mean those new podcasts won't appear on their other devices?

Based on my tests it syncs just fine (as per the PR description)